docs/api.rst

```diff
@@ -1,126 +1,197 @@
 
-.. automodule:: aospy.calc
+.. autoclass:: aospy.model.Model
     :members:
     :undoc-members:
-    :show-inheritance:
 
-aospy.constants module
-----------------------
+    .. automethod:: aospy.model.Model.__init__
 
-.. automodule:: aospy.constants
+Run
+---
+
+.. autoclass:: aospy.run.Run
     :members:
     :undoc-members:
-    :show-inheritance:
 
-aospy.data_loader module
-------------------------
+    .. automethod:: aospy.run.Run.__init__
 
-.. automodule:: aospy.data_loader
+DataLoaders
+===========
+
+The `Proj`, `Model`, and `Run` objects rely on a helper "data loader"
+to specify how to find their underlying data that is saved on disk.
+This mapping of variables, time ranges, and potentially other
+parameters to the location of the corresponding data on disk can
+differ among modeling centers or even between different models at the
+same center.
+
+Currently supported data loader types are `DictDataLoader`,
+`NestedDictDataLoader`, and `GFDLDataLoader` Each of these inherit
+from the abstract base `DataLoader` class.
+
+.. autoclass:: aospy.data_loader.DataLoader
     :members:
     :undoc-members:
-    :show-inheritance:
 
-aospy.find_obj module
----------------------
+    .. automethod:: aospy.data_loader.DataLoader.__init__
 
-.. automodule:: aospy.find_obj
+.. autoclass:: aospy.data_loader.DictDataLoader
     :members:
     :undoc-members:
-    :show-inheritance:
 
-aospy.model module
-------------------
+    .. automethod:: aospy.data_loader.DictDataLoader.__init__
 
-.. automodule:: aospy.model
+.. autoclass:: aospy.data_loader.NestedDictDataLoader
     :members:
     :undoc-members:
-    :show-inheritance:
 
-aospy.operator module
----------------------
+    .. automethod:: aospy.data_loader.NestedDictDataLoader.__init__
 
-.. automodule:: aospy.operator
+.. autoclass:: aospy.data_loader.GFDLDataLoader
     :members:
     :undoc-members:
-    :show-inheritance:
 
-aospy.proj module
------------------
+    .. automethod:: aospy.data_loader.GFDLDataLoader.__init__
 
-.. automodule:: aospy.proj
+Variables and Regions
+=====================
+The `Var` and `Region` classes are used to represent, respectively,
+physical quantities the user wishes to be able to compute and
+geographical regions over which the user wishes to aggregate their
+calculations.
+
+Whereas the `Proj` - `Model` - `Run` hierarchy is used to describe the
+data resulting from particular model simulations, `Var` and `Region`
+represent the properties of generic physical entities that do not
+depend on the underlying data.
+
+Var
+---
+
+.. autoclass:: aospy.var.Var
     :members:
     :undoc-members:
-    :show-inheritance:
 
-aospy.region module
--------------------
+    .. automethod:: aospy.var.Var.__init__
 
-.. automodule:: aospy.region
+Region
+------
+
+.. autoclass:: aospy.region.Region
     :members:
     :undoc-members:
-    :show-inheritance:
 
-aospy.run module
-----------------
+    .. automethod:: aospy.region.Region.__init__
+
+Calculations
```

Comment:
If you have time, could you also flesh out the ```CalcInterface``` doc-string?  Currently one needs to be familiar with those arguments to create ```Calc``` objects.